### ISSUE TITLE
Why is node 16 required?

### ISSUE THREAD

**Issue by rijk** [breadadams/scroller-motion#17]
Props on the 1.0.0 release 👏

While installing it on my Next.js project, I got some errors as the maximum node version supported on Vercel is 14.

```
npm WARN EBADENGINE Unsupported engine {
--
15:50:41.491 | npm WARN EBADENGINE   package: 'scroller-motion@1.0.0',
15:50:41.491 | npm WARN EBADENGINE   required: { node: '>= 16' },
15:50:41.491 | npm WARN EBADENGINE   current: { node: 'v14.19.0', npm: '8.3.1' }
15:50:41.491 | npm WARN EBADENGINE }
```

I think it should be ok to lower this requirement?

**Comment by breadadams** [comment]
Thanks @rijk 🙂, and thanks for reporting the issues. I'll check them out this week.

**Comment by breadadams** [comment]
Hey @rijk, should be resolved in the `1.1.0` release, thanks again!
